test_scripts/Resumption/Handling_errors_from_HMI/050_resume_IVD_subscription_after_erroneous_response_all_moduleTypes.lua

```diff
@@ -0,0 +1,104 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description:
+-- In case:
+-- 1. Subscriptions for all interior vehicle data types are added
+-- 2. Unexpected disconnect and Reconnect are performed
+-- 3. App reregisters with actual HashId
+-- 4. HMI responds with erroneous resultCode for one interior vehicle data type
+-- SDL does:
+-- 1. process unsuccess response from HMI
+-- 2. remove already restored data
+-- 3. respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local Function ]]
+local function getValue(pModuleType, pSubscribe)
```

Comment:
@GetmanetsIrina I would propose to rename this function and move it to common, because it is used in multiple scripts